### ISSUE TITLE
mdAutocomplete, mdMenu : Matched Items List gets placed behind menu when using md-autocomplete inside of  a md-menu directive.

### ISSUE THREAD

**Issue by scottkipfer** [angular/material#7424]
Issue: When I place a md-autocomplete directive inside of a md-menu directive, the matched items list gets put behind the menu making it impossible to see or select the first items.

<img width="719" alt="screen shot 2016-03-05 at 10 53 34 pm" src="https://cloud.githubusercontent.com/assets/5027577/13552374/b7e0b684-e325-11e5-9a35-ecb029d29c21.png">

I noticed that .md-open-menu-container and .md-virtual-repeat-container.md-autocomplete-suggestions-container both have z-index: 100.  

changing the z-index of .md-virtual-repeat-container.md-autocomplete-suggestions-container to 101 seems to fix it.

Version: 1.0.5 && 1.0.6
Browser: checked in Chrome and FF
Reproduced: [code pen](http://codepen.io/richardnixonthethird/pen/MyadpZ)

**Comment by DevVersion** [comment]
Yes, can reproduce that, but I'm not about increasing the z-index for the suggestion box.
@EladBezalel have you any thoughts?

**Comment by EladBezalel** [comment]
@scottkipfer are you trying to create a searchable select box?
if so please take a look at #7385

Besides that, i think we should make an effort and reorganize our z-indices

**Comment by scottkipfer** [comment]
@EladBezalel actually no,  In my actual application the auto-complete boxes are for different models,  I just used the same auto-complete in the demo pen to quickly reproduce the issue.  

But the searchable select box does look awesome and I could see other places in my app where I could use that.

As a work around I just used my own css to override the  .md-virtual-repeat-container.md-autocomplete-suggestions-container z-index to 101,  but I'm not sure if it breaks anything else.

Thanks for looking into this.
